src/Illuminate/Database/Console/MonitorCommand.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace Illuminate\Database\Console;
+
+use Illuminate\Contracts\Events\Dispatcher;
+use Illuminate\Database\ConnectionResolverInterface;
+use Illuminate\Database\Events\DatabaseBusy;
+use Symfony\Component\Console\Attribute\AsCommand;
+
+#[AsCommand(name: 'db:monitor')]
+class MonitorCommand extends AbstractDatabaseCommand
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'db:monitor
+                {--databases= : The names of the databases to monitor}
```

Comment:
I think this should an argument (and not an option), that should inspect the `default` database my default.

Reply:
For this, we took the lead from existing commands such as `migrate`, `show:model` and `schema:dump` so I think it probably makes sense to leave it as an option.

Reply:
Changed for this command so that it matches `queue:monitor`.